src/Microsoft.ML.Featurizers/TimeSeriesImputer.cs

```diff
@@ -83,6 +83,16 @@ public static class TimeSeriesImputerExtensionClass
     /// | Output column data type | All Types |
     /// | Exportable to ONNX | No |
     ///
+    /// The TimeSeriesImputer imputes missing rows and column data per grain (category), based on the dates in the date column. This operation needs to happen to every column in the IDataView,
+    /// If you "filter" a column using the filterColumns and filterMode parameters, if a row is imputed the default value for that type will be used.
+    /// Currently only float/double/string columns are supported for imputation strategies, and an empty string is considered "missing" for the
+    /// purpose of this estimator. A new column is added to the schema after this operation is run. The column is called "IsRowImputed" and is a
+    /// boolean value representing if the row was created as a result of this operation or not.
+    ///
+    /// NOTE: It is not recommended to chain this multiple times. If a column is filtered, the default value is placed when a row is imputed, and the
+    /// default value is not null. Thus any other TimeSeriesImputers will not be able to replace those values anymore causing essentially a very
+    /// computationally expensive NO-OP.
+    ///
     /// The <xref:Microsoft.ML.Transforms.TimeSeriesImputerEstimator> is not a trivial estimator and needs training.
```

Comment:
This xref is not resolving. Do we even need this line? We have the information in the table already

Reply:
We don't need that line. I'll go ahead and remove it.